scss/mixins/_forms.scss

```diff
@@ -49,7 +49,7 @@
 
       .form-control-color {
         @include form-validation-state-selector($state) {
-          width: add($form-color-width, $input-height-inner);
+          width: calc(#{$form-color-width} + #{$form-feedback-icon-size} + .8125rem); // stylelint-disable-line function-disallowed-list
```

Comment:
I think the icon isn't well centered. I found 2 solutions:
- First one uses `calc(#{$form-color-width} + #{$form-feedback-icon-size} + 1.0625rem)` and sets the padding-right to `1.25em + 1.25rem`
- The second one uses `calc(#{$form-color-width} + #{$form-feedback-icon-size} + .5rem)` and sets `background-position: right $input-height-inner-quarter top subtract($input-height-inner-quarter, 2px);`

I don't know which one's better here.

Reply:
We worked together for a better version. Still not perfect, but due to previous code.